### PR TITLE
Add bg and imagesCircle options 

### PR DESCRIPTION
**Pull request by chibicode**
Adds the following options (no changes to the form https://og-image.vercel.app/):

### Before

https://og-image-git-bg-rounded-options.vercel.sh/Before.png?theme=light&md=1&fontSize=100px&images=https%3A%2F%2Fassets.vercel.com%2Fimage%2Fupload%2Ffront%2Fassets%2Fdesign%2Fvercel-triangle-black.svg

### After (adding `&bg=plain`)

https://og-image-git-bg-rounded-options.vercel.sh/After.png?theme=white&md=1&fontSize=100px&images=https%3A%2F%2Fassets.vercel.com%2Fimage%2Fupload%2Ffront%2Fassets%2Fdesign%2Fvercel-triangle-black.svg&bg=plain

### Before

https://og-image-git-bg-rounded-options.vercel.sh/**Hello**%20World.png?images=https%3A%2F%2Fassets.vercel.com%2Fimage%2Fupload%2Ffront%2Fassets%2Fdesign%2Fvercel-triangle-black.svg&images=https%3A%2F%2Fcdn.jsdelivr.net%2Fgh%2Fremojansen%2Flogo.ts%40master%2Fts.svg&fontSize=100px&theme=light&md=1

### After (adding `&imagesCircle=0&imagesCircle=1`)

https://og-image-git-bg-rounded-options.vercel.sh/**Hello**%20World.png?images=https%3A%2F%2Fassets.vercel.com%2Fimage%2Fupload%2Ffront%2Fassets%2Fdesign%2Fvercel-triangle-black.svg&images=https%3A%2F%2Fcdn.jsdelivr.net%2Fgh%2Fremojansen%2Flogo.ts%40master%2Fts.svg&fontSize=100px&theme=light&md=1&imagesCircle=0&imagesCircle=1

### CODE REVIEW

File: api/_lib/parser.ts [hunk 1]
```diff
@@ -51,9 +53,10 @@ function getArray(stringOrArray: string[] | string | undefined): string[] {
 }
 
 function getDefaultImages(images: string[], theme: Theme): string[] {
-    const defaultImage = theme === 'light'
-        ? 'https://assets.vercel.com/image/upload/front/assets/design/vercel-triangle-black.svg'
-        : 'https://assets.vercel.com/image/upload/front/assets/design/vercel-triangle-white.svg';
+    const defaultImage =
+        theme === 'light'
+            ? 'https://assets.vercel.com/image/upload/front/assets/design/vercel-triangle-black.svg'
+            : 'https://assets.vercel.com/image/upload/front/assets/design/vercel-triangle-white.svg';
```
Comment (chibicode) [inline api/_lib/parser.ts:59]: prettier only change.

File: api/_lib/parser.ts [hunk 2]
```diff
@@ -35,6 +35,8 @@ export function parseRequest(req: IncomingMessage) {
         images: getArray(images),
         widths: getArray(widths),
         heights: getArray(heights),
+        imagesCircle: getArray(imagesCircle).map((x) => x === '1' || x === 'true'),
+        hideDots: hideDots === '1' || hideDots === 'true'
```
Comment (styfle) [inline api/_lib/parser.ts:39]: I could see this being extended in the future, so lets use `bg=plain` and then we could have `bg=dots` be the default.

The reason is I could see that we might want something like `bg=transparent` or maybe other patterns in the future.

Comment (chibicode) [reply api/_lib/parser.ts:39]: @styfle will do!

File: api/_lib/template.ts [hunk 3]
```diff
@@ -77,6 +80,10 @@ function getCss(theme: string, fontSize: string) {
         margin: 0 75px;
     }
 
+    .logo-rounded {
+        border-radius: 9999px;
```
Comment (styfle) [inline api/_lib/template.ts:84]:
```suggestion
        border-radius: 50%;
```

I think this is a common pattern

Comment (chibicode) [reply api/_lib/template.ts:84]: @styfle yeah I think both is fine in this case. It has different results for non-square images, but I don't think that matters here.
https://github.com/tailwindlabs/tailwindcss/issues/1949

File: api/_lib/parser.ts (hunk 2 above)
Comment (chibicode) [reply api/_lib/parser.ts]: @styfle acc6b83

**Review by styfle — APPROVED** [review]
Thanks!

**Comment by styfle** [comment]
Closing since we discussed that these changes will live in a different repo

**Pull request closed without merging**